src/test/scala/kevinlee/sbt/io/Gens.scala

```diff
@@ -0,0 +1,41 @@
+package kevinlee.sbt.io
+
+import java.nio.charset.StandardCharsets
+import java.security.SecureRandom
+
+import hedgehog._
+import kevinlee.test.data.{Content, Names, NamesAndContent}
+
+import scala.util.Random
+
+/**
+  * @author Kevin Lee
+  * @since 2019-02-23
+  */
+object Gens {
+
+  val random: Random =
+    new Random(new SecureRandom("Gens".getBytes(StandardCharsets.UTF_8)))
+
+
+  def genFilenamesAndContent: Gen[(List[String], String)] = for {
+    names <- Gen.string(Gen.alphaNum, Range.linear(1, 5)).list(Range.linear(1, 5))
+    content <- Gen.string(Gen.unicode, Range.linear(5, 50))
+  } yield (names, content)
+
+  def genFilenamesAndContentWithFirstUniqueName: Gen[List[NamesAndContent]] =
+    Gens.genFilenamesAndContent
+      .list(Range.linear(1, 5))
+      .map { namesList =>
+        namesList.map { case (names, content) =>
+          val randomNumber = random.nextInt()
```

Comment:
Why the random number, you're in `Gen`, `Gen.int(...)`?

Reply:
Oh, that's actually something I need to ask you. 😆 So I need `List[List[String]]` but want to make sure that the first String in every inner `List` is unique.
If I don't add that kind of random thing, I sometimes get something like
```scala
List(
  List(
    "abc", "blah1", "blas2"
  )
, List(
    "abc", "lojlashf" 
  )
, List(
    "regn", "sjlskfa", "zzz", "eklj"
  )
)
```
Aa you can see the first and second Lists have the same String (i.e. `"abc"`)

How can I make sure each List has unique first element?

Reply:
If you just want the exact same thing you have then generate a random int using hedgehog, perhaps with `Range.constant` to avoid more likely clashes in the smaller/first tests.

```scala
.flatMap { namesList =>
  namesList.flatMap { case (names, content) =>
    Gen.int(Range.constant(0, Int.MinValue, Int.MaxValue)).flatMap(randomNumber =>
``` 

But I personally would use something _guaranteed_ to be unique, like the index of the element in the list:

```scala
gen.list(Range.lineaer(1, 5)).map(_.zipWithIndex(x => x._1 + "_" + x._2.toString))
```

Reply:
Ah... `zipWithIndex` is what I need. Thanks a lot Charles!